modules/tinymce/src/core/demo/ts/demo/FullDemo.ts

```diff
@@ -47,7 +47,7 @@ export default () => {
       { title: 'Some class', value: 'class-name' }
     ],
     importcss_append: true,
-    height: 400,
+    height: 200,
```

Comment:
Could you please put this back to its original value?
```suggestion
    height: 400,
```

Reply:
updated